### PR TITLE
OCD-4528: Add analytics for ONC Organizations 

### PR DESCRIPTION
**Pull request by mattystank**

### CODE REVIEW

File: src/app/pages/organizations/onc-organizations/onc-organizations.jsx [hunk 1]
```diff
@@ -79,8 +80,7 @@ function ChplOncOrganizations() {
   const userQuery = useFetchUsersAtAcb(orgs.find((org) => org.id === activeId), orgType);
   const roles = ['chpl-onc-acb'];
   const classes = useStyles();
-
-  useEffect(() => {
+  const organization = useEffect(() => {
```
Comment (andlar) [inline src/app/pages/organizations/onc-organizations/onc-organizations.jsx]: This doesn't work. I don't know what you're trying to do, but `useEffect` doesn't return anything, so can't be used to set the organization to any particular value

File: src/app/components/onc-organization/onc-organization-view.jsx [hunk 2]
```diff
@@ -138,7 +139,14 @@ function ChplOncOrganizationView(props) {
...
               aria-label={`Edit ${organization.name} Information`}
               id="organization-component-edit"
-              onClick={edit}
+              onClick={() => {
+                ReactGA.event('test',
+                  {
+                    category: 'ONC Organizations',
+                    label: `${organization.name}`,
+                  });
+                  edit(true);
```
Comment (andlar) [inline src/app/components/onc-organization/onc-organization-view.jsx]:
```suggestion
                  edit();
```

File: src/app/pages/organizations/onc-organizations/onc-organizations.jsx [hunk 3]
```diff
@@ -241,10 +250,9 @@ function ChplOncOrganizations() {
                       color="primary"
                       onClick={() => setIsCreating(true)}
                       endIcon={<AddIcon />}
-                      id="create-new-organization"
-                    >
+                      id="create-new-organization">
```
Comment (andlar) [inline src/app/pages/organizations/onc-organizations/onc-organizations.jsx]:
```suggestion
                      id="create-new-organization"
                    >
```

File: src/app/pages/organizations/onc-organizations/onc-organizations.jsx [hunk 4]
```diff
@@ -241,10 +250,9 @@ function ChplOncOrganizations() {
...
                       Create
-                    </Button>
+                      </Button>
```
Comment (andlar) [inline src/app/pages/organizations/onc-organizations/onc-organizations.jsx]:
```suggestion
                    </Button>
```

**Review by andlar — CHANGES_REQUESTED** [review]

**Review by andlar — APPROVED** [review]

**Review by kekey1 — APPROVED** [review]

**Review by tmy1313 — APPROVED** [review]

**Pull request merged**